services/validators.go

```diff
@@ -146,5 +147,21 @@ func ValidateServiceAgreement(sa models.ServiceAgreement, store *store.Store) er
 		fe.Add(fmt.Sprintf("Service agreement job spec error: Job spec validation: %v", err))
 	}
 
+	untilEndAt := time.Until(sa.Encumbrance.EndAt.Time)
+	fmt.Print("MaximumServiceDuration ", config.MaximumServiceDuration,
```

Comment:
✂️ this line.

Don't think we want to leave `fmt.Print` statements in the validation code. If anything, we would do `logger.Debug[fw]`.